src/components/compassSlider.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useRef, useEffect } from "react";
+import { describeSlice } from "../utils/utils";
+
+type Props = {
+  radius: number;
+  angle: number;
+  setAngle: (value: number) => void;
+  min?: number;
+  max?: number;
+};
+
+const CompassSlider = ({
+  radius = 60,
+  angle,
+  setAngle,
+  min = 0,
+  max = 359.999,
```

Comment:
change to 360

Reply:
if i change it to zero it will mark all of it as red i am pretty sure